.circleci/config.yml

```diff
@@ -7,16 +7,20 @@ workflows:
   build:
     jobs:
       - elixir/build_test:
+          cache-version: 2
```

Comment:
Is this one still required or have you bumped it before Tuesday? I believed I've "fixed" the CI by disallowing less strict cache usage

Reply:
I bumped it today as I was having some credo problems and I believie the bump fixed them, I can double check whether that's the case

Reply:
Oh, as far as I can see in the commits history, the only cache version bumps where the ones from 2 weeks ago - back then, an old version of ORB was used

Reply:
Oh, I must've confused something